### DISCUSSION TITLE
Problema añadiendo archivos de segunda entrega

### DISCUSSION THREAD

**Discussion by gaborszabo17** [Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso#19] [Dudas del curso]
Hola, Jairo. ¿Cómo estás?
Te comento que he estado teniendo el siguiente error a la hora de hacer el comando git add act2.py ... en la terminal. Pude publicarlos en el remoto a partir de las opciones que tiene VCS, pero me gustaría entender qué estaba haciendo mal yo :(.

Acá añado un screenshot del mensaje

![image](https://user-images.githubusercontent.com/123516687/216859030-85db9235-c3b1-46b4-bce7-699e93e1e017.png)

**Comment by jairomelo** [comment]
Hola,

En realidad no estás haciendo nada "mal". El error se debe a que no encuentra el archivo con la ruta indicada. Es decir, git está intentando ubicar el archivo en la raíz del directorio './act.py' en lugar de la ruta completa 'estudiantes/Christian David Baumgartner/Actividad 2/act2.py'

Es un error en las instrucciones de la actividad, una disculpa.

La mejor opción en este caso debería ser:
```
git add .
git commit -m "mensaje del commit"
git push -u origin tubranch
```

Muchas gracias por publicar tu duda por este espacio para que los demás en @Taller-Abierto-de-Humanidades-Digitales/fundamentos-programacion puedan solucionar el error.
